Source/WebCore/html/ColorInputType.cpp

```diff
@@ -269,7 +265,7 @@ void ColorInputType::handleDOMActivateEvent(Event& event)
     event.setDefaultHandled();
 }
 
-void ColorInputType::showPicker() 
+void ColorInputType::showPicker()
```

Comment:
What changed here? It looks identical. Could you try and drop whatever the change was?

Reply:
There was an extra space after the closing parenthesis. I can revert it but I guess it was the linter (?)

Reply:
Ah okay (GitHub diff doesn't show that very well) I guess leave it changed else it'll just end up being changed by the next change to the file.